### ISSUE TITLE
Symbol's value as variable is void: company-backends

### ISSUE THREAD

**Issue by xuehao** [proofit404/company-anaconda#9]
I can't configure it.

**Comment by proofit404** [comment]
Does this work?
```lisp
(eval-after-load "company"
 '(progn
   (add-to-list 'company-backends 'company-anaconda)))
```

**Comment by xuehao** [comment]
@proofit404 Nicely!! It works find. How do you do it, it's amazing!

**Comment by proofit404** [comment]
This works because of deferred code evaluation. `add-to-list` will be executed only after loading `company` library so `company-backends` variable will be defined.
